model/pail_storage.go

```diff
@@ -82,6 +82,42 @@ func (t PailType) Create(ctx context.Context, env cedar.Environment, bucket, pre
 	return b, nil
 }
 
+// CreatePresto returns a Pail Bucket backed by PailType specifically for
+// buckets in our Presto ecosystem.
+func (t PailType) CreatePresto(ctx context.Context, env cedar.Environment, prefix, permissions string, compress bool) (pail.Bucket, error) {
+	conf := &CedarConfig{}
+	conf.Setup(env)
+	if err := conf.Find(); err != nil {
+		return nil, errors.Wrap(err, "getting application configuration")
+	}
+
+	switch t {
+	case PailS3:
+		opts := pail.S3Options{
+			Name:          conf.Bucket.PrestoBucket,
+			Prefix:        prefix,
+			Region:        defaultS3Region,
+			Permissions:   pail.S3Permissions(permissions),
+			Credentials:   pail.CreateAWSCredentials(conf.Bucket.AWSKey, conf.Bucket.AWSSecret, ""),
```

Comment:
Since the IAM role is associated with the app server host, is it still necessary to set these credentials?

Reply:
The IAM role is associated with the cedar IAM user, not the app server host, in this case.